iphone/Classes/TiUITabGroup.m

```diff
@@ -370,25 +370,52 @@ - (void)tabBarController:(UITabBarController *)tabBarController didEndCustomizin
 
 - (void)setTabBarVisible_:(id)value
 {
-  BOOL visible = [TiUtils boolValue:value];
+  bool isIOSVersionLowerThan18 = [TiUtils isIOSVersionLower:@"18.0"];
```

Comment:
The proper ObjC type is `BOOL`, not the c type `bool`. Also rename it to `isiOS17OrLower`